src/client/api/SConscript

```diff
@@ -22,17 +23,18 @@ def scons():
     Export('libdaos_tgts')
 
     if prereqs.client_requested():
-        libdaos= daos_build.library(env, 'daos', libdaos_tgts,
+        libdaos = daos_build.library(env, 'daos', libdaos_tgts,
                                      SHLIBVERSION=API_VERSION,
                                      LIBS=['daos_common'])
+        env.Requires(libdaos, [common])
```

Comment:
Minor nit
Might help a little if the name were client_common or daos_common rather than just common.